speech/grpc/bin/speech-sample-async.sh

```diff
@@ -15,4 +15,4 @@
 
 SRC_DIR=$(cd "$(dirname "$0")/.."; pwd)
 java -cp ${SRC_DIR}/target/grpc-sample-1.0-jar-with-dependencies.jar \
```

Comment:
This allows you to run the code from anywhere. Otherwise, you have to be in the correct directory to run the code. This is better!